### ISSUE TITLE
Character: teachable agent mode

### ISSUE THREAD

**Issue by wekitecs** [WEKIT-ECS/MIRAGE-XR#365]
In GitLab by @wild on Apr 12, 2021, 11:44

The user controls where the alien walks to do a 'mission'. This may require the author to set target locations. Requires more discussion.

**Comment by wekitecs** [comment]
In GitLab by @wild on Sep 10, 2021, 12:57

I have an idea for this: we could add a checkbox: [ ] teachable agent, which adds a gaze cursor which upon airtap makes the character model move to the target location. Then we could use a similar mechanism as with Pick&Place: the user has to lead the character to each location of the path (in the order of the path) to trigger the move to next action step?

**Comment by fwild** [comment]
I believe significant parts of this idea we have now already captured in other tickets: https://github.com/WEKIT-ECS/MIRAGE-XR/issues/945 and the character model as such. Moreover, we have added the possibility to add path steps for the character, effectively implementing the idea describe above of moving to target locations, but only via edit mode (not playmode as user generated content). 

There is still something about teachable agents that is intriguing, but we don't have this clearly formulated - therefore closing this ticket to make room for a new one.
